easybuild/easyconfigs/c/Caffe/Caffe-1.0-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,67 @@
+# -*- mode: python; -*-
+# EasyBuild reciPY for Caffe as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 UL HPC
+# Authors::   UL HPC Team <hpc-sysadmins@uni.lu>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'Caffe'
+version = '1.0'
+
+# cudaversion = '7.5.18'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/BVLC/caffe'
+description = """
+Caffe is a deep learning framework made with expression, speed,
+and modularity in mind. It is developed by the Berkeley Vision
+and Learning Center (BVLC) and community contributors.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+# toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/BVLC/caffe/archive/']
+
+builddependencies = [
+    ('CMake', '3.8.1'),
+]
+
+protobuf_ver = '3.3.0'
+
+# See http://caffe.berkeleyvision.org/installation.html
+dependencies = [
+    ('Boost', '1.63.0', '-Python-%(pyver)s'),
+    ('protobuf', protobuf_ver),
+    ('protobuf-python', protobuf_ver, '-Python-%(pyver)s'),
+    ('glog', '0.3.4'),
+    ('gflags', '2.1.2'),
+    ('HDF5', '1.8.18'),
+    # ('OpenCV', '3.3.0', '-Python-%(pyver)s'),
+    # ('CUDA', cudaversion, '', True),
+    # ('cuDNN', '4.0', '', True),
+    ('LMDB', '0.9.18'),
+    ('LevelDB', '1.18'),
+    ('snappy', '1.1.6'),
+    ('Python', '2.7.13'),
+    ('scikit-image', '0.13.0', '-Python-%(pyver)s')
+]
+
+configopts = '-DBLAS=mkl -DUSE_OPENCV=0 -DUSE_MKL2017_AS_DEFAULT_ENGINE=1 -DCPU_ONLY=1'
+
+modextrapaths = {'PYTHONPATH': ['python']}
+
+sanity_check_paths = {
+    'files': ['bin/caffe'] +
+             ['lib64/%s' % x for x in ['libcaffe.so', 'libproto.a']] +
```

Comment:
Please use `SHLIB_EXT` for shared library filename extensions.

Reply:
added in 43f2d57